keyboards/idb_60/idb_60.h

```diff
@@ -0,0 +1,33 @@
+#ifndef _IDB_60_H
+#define _IDB_60_H
+
+#include "quantum.h"
+#include "led.h"
+
+inline void _idb_60_caps_led_on(void)      { DDRC |=  (1<<5); PORTC &= ~(1<<5); }
+inline void _idb_60_esc_led_on(void)    	{ DDRC |=  (1<<4); PORTC &= ~(1<<4); }
+
+inline void _idb_60_caps_led_off(void)     { DDRC &= ~(1<<5); PORTC &= ~(1<<5); }
+inline void _idb_60_esc_led_off(void)   	{ DDRC &= ~(1<<4); PORTC &= ~(1<<4); }
+
+#define LAYOUT( \
```

Comment:
The spacing on this layout macro is inconsistent. It'll work, but it doesn't look polished.

The electrical matrix (Lines 20-29) also only defines 7 positions for rows 1-9, even though `MATRIX_COLS` is 8. Obviously `K07` is the only position on that column, and the code still works this way.

I'd prefer if these issues were tidied up, but you don't absolutely have to.